packages/combobox/index.tsx

```diff
@@ -18,11 +19,12 @@ export interface ComboboxProps {
   renderOption?: (option: ComboboxOptionValue) => ReactNode
   options: ComboboxOptionValue[]
   openOnFocus?: boolean
-  onInputChange?: (e: React.ChangeEvent<HTMLInputElement>) => void
-  onInputBlur?: (e: React.FocusEvent<HTMLInputElement>) => void
+  inputProps: Optional<ComboboxInputProps, 'onChange'>
```

Comment:
Sorry if I just don't have context but am curious why not make it optional on the type itself? Is there another instance where it's required?

Reply:
That is a really great point! -- I'll take a look, I think it probably can be optional at base